enterprise_access/apps/api_client/braze_client.py

```diff
@@ -28,3 +29,48 @@ def __init__(self):
             api_url=settings.BRAZE_API_URL,
             app_id=settings.BRAZE_APP_ID
         )
+
+    def generate_mailto_link(self, emails):
```

Comment:
why does this need to be in the braze client?

Reply:
[long] need CSV list, easier than template. braze lib better than random task method hanging out.